server.go

```diff
@@ -0,0 +1,70 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"strings"
```

Comment:
I made a hacky tool to group imports correctly FWIW.

    go get github.com/rogpeppe/sortimports

If you run "sortimports ./..." it's just like "go fmt ./..." except that imports
will be grouped according to our usual conventions.